src/umockdev.vala

```diff
@@ -75,6 +75,10 @@ public class Testbed: GLib.Object {
         this.dev_fd = new HashTable<string, int> (str_hash, str_equal);
         this.dev_script_runner = new HashTable<string, ScriptRunner> (str_hash, str_equal);
 
+        this.worker_ctx = new MainContext();
+        this.worker_loop = new MainLoop(this.worker_ctx);
+        this.worker_thread = new Thread<void> ("umockdev-testbed-worker", this.worker_run);
```

Comment:
This makes sense, but I think this commit should be squashed into one of the next ones that actually *uses* the worker thread. Especially because the next commit rewrites/reverts most of the changes here.